index.js

```diff
@@ -331,12 +338,12 @@ ShareDbMongo.prototype.getSnapshot = function(collectionName, id, fields, option
     if (err) return callback(err);
     var query = {_id: id};
     var projection = getProjection(fields, options);
-    var request = createRequestForMiddleware(options, collectionName);
+    var request = createRequestForMiddleware(fields, options, collectionName);
     request.query = query;
     self._middleware.trigger(MiddlewareHandler.Actions.beforeSnapshotLookup, request, function(middlewareErr) {
       if (middlewareErr) return callback(middlewareErr);
 
-      collection.find(request.query).limit(1).project(projection).next(function(err, doc) {
+      collection.find(request.query, request.queryOptions).limit(1).project(projection).next(function(err, doc) {
```

Comment:
Hm... this appears to be a new feature not mentioned in the PR description, being able to pass options to `Collection#find`.

I can see how this might be useful, though I'd ask:
1. For it to be mentioned in the PR description, and documented in the README.
2. Perhaps a more specific name? E.g. `findOptions`, since it's being used solely for `find()` and that's the [type name in the Mongo driver](https://mongodb.github.io/node-mongodb-native/4.0/interfaces/findoptions.html).

Reply:
Sorry missed it in the description...

Reply:
I'm pretty sure we use `find()` in more places than the places you've added `queryOptions` here, so I'm not even sure `findOptions` is specific enough? What was the use-case for this?

Reply:
@alecgibson We want our middleware to be able to modify the options passed to the Mongo driver for all queries related to looking up and updating (?) snapshots when submitting an op.

Reply:
Then shouldn't we be passing these options everywhere we call `find()`?

Reply:
@alecgibson Can you give an example of a place where it's missing in this scenario? We technically only need it for the snapshot lookup itself. This is for tombstone exclusion, since our Mongo middleware by default will exclude tombstones even on ID lookups.

Reply:
@alecgibson I see what you mean, added findOptions to missing find calls.